src/main/java/de/hysky/skyblocker/skyblock/waypoint/MythologicalRitual.java

```diff
@@ -213,6 +238,10 @@ private static class GriffinBurrow extends Waypoint {
         private Vec3d[] echoBurrowDirection;
         @Nullable
         private Vec3d[] echoBurrowLine;
+        @Nullable
+        private BlockPos estimatedPos;
+        @Nullable
+        private Line nextLine; // The echo line could be stored, but it is easily computed so eh
```

Comment:
Call `estimatedPos` -> `nextBurrowEstimatedPos`. Also maybe call `nextLine` -> `nextBurrowLine4Estimation` or something and add a comment explaning what it does, since `nextLine` is going to get confusing.